relayer/path.go

```diff
@@ -200,12 +200,12 @@ func (p *Path) QueryPathStatus(ctx context.Context, src, dst *Chain) *PathWithSt
 	eg = new(errgroup.Group)
 	eg.Go(func() error {
 		var err error
-		srcCs, err = src.ChainProvider.QueryClientStateResponse(srch, src.ClientID())
+		srcCs, err = src.ChainProvider.QueryClientStateResponse(ctx, srch, src.ClientID())
```

Comment:
This should be using a context derived from errgroup.WithContext. There are a few other occurrences in this PR.

Reply:
I'll go back through and get these squared away.